board/app/models/thread.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+class Thread extends AppModel{
+
+//Minimum Length
+const MIN_TITLE_LENGTH = 1;
+
+//Maximum Length
```

Comment:
I think the variable names are already descriptive enough, so no need to put in comments to the const vars :bow: (please apply to all)